### ISSUE TITLE
Regression exception -- Allowing vertical metrics fix

### ISSUE THREAD

**Issue by xconsau** [xconsau/KumbhSans#10]
Hello @davelab6 @RosaWagner,

As already pointed out in [this issue](https://github.com/xconsau/KumbhSans/issues/3), the current version of Kumbh Sans has improper vertical metrics. I have fixed this in an upcoming Variable Font update. But then it falls under the vertical metrics regression check:

<pre>>> com.google.fonts/check/vertical_metrics_regressions
    Check if the vertical metrics of a family are similar to the same family hosted on Google Fonts.</pre>

Below are the comparison images of HTML rendering of original vs updated versions:

![original](https://user-images.githubusercontent.com/11784237/118844986-0377b680-b8e9-11eb-84db-d806289486f3.jpg)
![updated](https://user-images.githubusercontent.com/11784237/118845016-0a9ec480-b8e9-11eb-94a3-a5a7b81ab454.jpg)

As you can see, except for the <code>openTypeHheaAscender</code> and <code>openTypeOS2TypoAscender</code>, all other metrics are kept same as original one. If this change can be allowed as an exception, it will improve the appearance of this font.

**Comment by davelab6** [comment]
I think we should push the new v metrics; its clearly better.

**Comment by davelab6** [comment]
I see we added this only last August (https://github.com/google/fonts/blob/main/ofl/kumbhsans/KumbhSans-Regular.ttf) - @m4rc1e did our vertical metrics spec change since then?

**Comment by m4rc1e** [comment]
Nope. The new changes may make the highlighting appear centred. However, the line heights will change for win and mac in text edit and Word.

Since the family is newish, I don't mind us rolling this update.

**Comment by RosaWagner** [comment]
Uh, I thought it was older. 

The official checks for v-metrics doesn't include all our v-metrics recommendations yet (typoAscender > Agrave y-max, caps centred, typoAscender+typoDescender>UPM*1,2). So fontbakery speaking, the font was all right.

**Comment by xconsau** [comment]
> However, the line heights will change for win and mac in text edit and Word.

![lineheight](https://user-images.githubusercontent.com/11784237/119134115-5ae96400-ba5a-11eb-8fa6-1a23c6c85d4a.jpg)

Agreed. I also see that the HTML Buttons and Form input fields may need revised padding if Web Designers have applied unequal top/bottom padding to make it vertically centered.

> Since the family is newish, I don't mind us rolling this update.

That would be great.

**Comment by RosaWagner** [comment]
We can also keep the same linepsacing and center capitals within them. Would be less of a change.

**Comment by xconsau** [comment]
> We can also keep the same linepsacing and center capitals within them. Would be less of a change.

The TypoDescender value is already tight at -520. We only have scope in TypoAscender. From 2045 we can bring it down to 2020 or 2000 max (which is the top of Agrave). At 2020 or 2010, this condition is also satisfied &mdash; (TypoAscender > Agrave ymax, and TypoAscender + TypoDescender > UPM*1.2)

![lineheight2](https://user-images.githubusercontent.com/11784237/119158889-53cf4f80-ba74-11eb-80da-91c64b5bd2de.jpg)

So we can reduce around 35 units without causing clipping.

**Comment by davelab6** [comment]
That sounds good @xconsau
